### PR TITLE
Update Read the Docs links

### PR DESCRIPTION
**Pull request by l2dy**

### CODE REVIEW

File: src/storyboard/metadata.py [hunk 1]
```diff
@@ -1234,7 +1234,7 @@ def main():
     in https://docs.python.org/3/library/configparser.html).
 
     For more detailed explanations, see
-    http://storyboard.rtfd.org/en/stable/metadata-cli.html (or replace
+    https://storyboard.readthedocs.io/en/stable/metadata-cli.html (or replace
```
Comment (zmwangx) [inline src/storyboard/metadata.py]: This needs to be refilled to 72 columns.

File: src/storyboard/storyboard.py [hunk 2]
```diff
@@ -1068,7 +1068,7 @@ def main():
     wrapper script around the storyboard.storyboard if you'd like to.
 
     For more detailed explanations, see
-    http://storyboard.rtfd.org/en/stable/storyboard-cli.html (or replace
+    https://storyboard.readthedocs.io/en/stable/storyboard-cli.html (or replace
```
Comment (zmwangx) [inline src/storyboard/storyboard.py]: Same here.

**Review by zmwangx — CHANGES_REQUESTED** [review]
Thanks. Two comments.

File: src/storyboard/metadata.py (hunk 1 above)
Comment (l2dy) [reply src/storyboard/metadata.py]: Fixed.

**Comment by zmwangx** [comment]
Committed in 34dba8f145caaa0f6fb0a529ecad5aec7c6f7eef. Thanks for your contribution!

**Pull request closed without merging**